src/compiler/types.ts

```diff
@@ -4325,7 +4325,7 @@ export interface SourceFile extends Declaration, LocalsContainer {
     // JS identifier-declarations that are intended to merge with globals
     /** @internal */ jsGlobalAugmentations?: SymbolTable;
 
-    /** @internal */ identifiers: ReadonlyMap<string, string>; // Map from a string to an interned string
+    /** @internal */ identifiers: Record<string, string>; // Map from a string to an interned string

```

Comment:
Just for if this comes out good, probably want `Record<string, string | undefined>`.